### ISSUE TITLE
Flag "include drop" also drops schema, but is not documented

### ISSUE THREAD

**Issue by Bourg** [dimitri/pgloader#825]
While migrating a MySQL database to PostgreSQL using pgloader, I discovered that the `include drop` argument to the `WITH` clause not only drops tables, but also the containing schema. According to the [relevant section in the documentation](https://pgloader.readthedocs.io/en/latest/ref/mysql.html#mysql-database-migration-options-with), the `include drop` argument "drops all the tables in the target PostgreSQL database whose names appear in the MySQL database," but it makes no mention of dropping the target schema as part of that mode. Could the documentation be updated to make this potentially destructive behavior more clear?

  - [ ] pgloader --version
  
    ```
    3.4.1
    ```
  
  - [ ] did you test a fresh compile from the source tree?
    
    No, but I did look in the current source code and found the section that drops and re-creates schemas.
    
  - [ ] did you search for other similar issues?

    Yes. None seemed to mention this behavior as it relates to the documentation.
    
  - [ ] how can I reproduce the bug?

   Run pgloader with `include drop` into a schema that contains other data. The data will be dropped, even if the table names don't interfere with the source table names.

  - [ ] How the data is different from what you expected, if relevant

    Based on the existing documentation, I did not expect a cascading schema drop to be executed as part of `include drop`.

**Comment by dimitri** [comment]
I wonder if that might be better classified as a code bug rather than a documentation bug. I guess you have an opinion on that?

**Comment by Bourg** [comment]
I figured it was best classified as a documentation issue since it seems to be expected behavior baked into the code with the ability to turn it on and off, though it sounds like the fact that this was default behavior in 3.4.1 was not intentional. I can definitely test it with a freshly compiled version.

Just for clarity, is the new expected behavior that `include drop` on its own should leave the containing schema unaffected, and that `drop schema` as an additional option to the `WITH` clause now enables the behavior that I experienced as a (potentially bugged) default in an older version?
